gravitee-apim-console-webui/src/management/api/policy-studio/debug/components/policy-studio-debug-inspector/policy-studio-debug-inspector.component.ts

```diff
@@ -203,13 +198,13 @@ export class PolicyStudioDebugInspectorComponent implements OnChanges {
     const errors = keys
       .filter((key) => key.startsWith('error.'))
       .sort(this.sort)
-      .map((key) => this.toErrorNode(key));
+      .map((key) => ({ key, value: this.inputDebugStep.output[key] || this.outputDebugStep.output[key] }));
```

Comment:
maybe `??` in case of empty string for inputDebugStep ? we want to keep it no ?  🤔